explore/index.html

```diff
@@ -24,7 +24,7 @@ <h2 class="header-red">OME for Scientists</h2>
         <div class="row">
             <img class="medium-3 medium-offset-1 columns" alt="OMERO for Scientists" src="{{ site.baseurl }}/img/icons/user-groups_scientists.svg">
             <div class="medium-8 columns">
-                <p>Sign up for our <a href="https://help.openmicroscopy.org/demo-server.html">demo server</a> to test out the functionality with your own data.</p>
+                <p>Sign up for our demo server to test out the functionality with your own data.</p>
```

Comment:
But now there is no link at all to go to.

I think in the meantime, we should point to http://qa.openmicroscopy.org.uk/registry/demo_account/